### ISSUE TITLE
Gates Narrative Reporting

### ISSUE THREAD

**Issue by gielinkg** [urbanbigdatacentre/ideamaps-models#130]
### ✅ Definition of Done
---

- [ ] 1. Define acceptance criteria.
- [ ] 2. Assess the need for a review process. If a review process is required, the issue states:
    1. Who is involved in the review?
    2. When will the review take place?
    3. Who is responsible for taking on the feedback?
    4. What additional tasks are involved and are they visible on the backlog?
- [ ] 3. Make progress and post updates.
- [ ] 4. Check off completed acceptance criteria.
- [ ] 5. Post links to digital outputs.
- [ ] 6. Note the value added to the product.

**Comment by gielinkg** [comment]
The full BMGF progress report is due on November 30th. There are separate financial requirements for this outlined in #129 

The items here are regarding the progress narrative. 

I have separated out progress and outlook sections for each WP (although I am still setting up the WP4 and 5 ones -- these will be there soon). These docs include what we wrote in last year's report, and have sections for updates about Year 2 progress and outlooks for Year 3. @andymithamclarke has already made updates for WP1 and @diegopajarito will be working on updates for WP2/3 as outlined in #127 

Below, is the location of draft documents which will be used to complete the full reporting template. 

**Folder here with all docs:** 
https://gla.sharepoint.com/:f:/s/IDEAMAPSEcosystem/Ej-SPZnkSEREuQ7fGXXdOfIBRJ-grR2cwCyTNMNKD9txtg?e=m1RdTy
